tests-upgrade/AutoRestUpgradeTest.ps1

```diff
@@ -265,7 +264,7 @@ if($TestName -ne $null -and ($TestName -ne ''))
     {
         foreach($blackTestName in $blackTestList)
         {
-            if(($fileDetail.Mode -eq 'd----') -and (!$fileDetail.Name.Startswith($blackTestName)))
+            if(($fileDetail.Mode.Startswith('d')) -and (!$fileDetail.Name.Startswith($blackTestName)))
```

Comment:
Please use file method instead of comparing mode string directly.

Reply:
Done，thanks